invite/test/e2e.test.ts

```diff
@@ -113,22 +122,20 @@ describe('end-to-end', () => {
             expect(body).toEqual({
               body: 'An invitation to join `test_org` has been sent to ' +
                 '`@someone`. I will update this thread when the invitation ' +
-                'accepted'
+                'is accepted.'
             });
             return true;
           })
           .reply(200);
 
         await triggerWebhook(probot, 'trigger_invite.issue_comment.created');
-        expect(await db('invites').select().first()).toEqual(recordedInvite);
+        const invites = await record.getInvites('someone');
+        expect(invites[0]).toMatchObject(recordedInvite);
```

Comment:
```typescript
expect(record.getInvites('someone')).resolves.arrayContaining(recordInvite);
```
(this way if invites is zero length or undefined or not an array, the test won't crash here)

Reply:
1) it would have to be `arrayContaining(expect.matchesObject(recordedInvite))` to ignore the extra `id` field
2) it should crash if the array is empty though :thinking:

Reply:
Crash ≠ fail >_<
(they result in the same thing, but you should try to write safe code wherever you can)

Reply:
Updated it to

```
expect(record.getInvites('someone')).resolves.toEqual([
  expect.objectContaining(recordedInvite)
]);
```